pants.toml

```diff
@@ -16,6 +16,11 @@ backend_packages = [
 # run this repository without worrying about Go being installed.
 pants_ignore.add = ["src/go"]
 
+# Opt in to future behaviors that will become the default in Pants v2.14.
+# These directives should be removed in Pants v2.14.x and higher.
+use_deprecated_directory_cli_args_semantics = false
```

Comment:
You might want to audit the readme to confirm that there are no commands that will change behavior with this.